src/pages/StudentExchange/StudentExchange.js

```diff
@@ -4,13 +4,18 @@ import Layout from 'components/Layout/Layout';
 const StudentExchange = ({ data }) => {
   if (!data) return null;
   else {
-    const { title, body } = data.student_exchange_page;
+    // console.log(data.builder)
+    let { builder } = data;
     return (
       <Layout>
-        <div className="studentexchange__container">
-          <h2 position="left" className="studentexchange__title">{title}</h2>
-          <div className="studentexchange__description" dangerouslySetInnerHTML={{__html: body}} />
-        </div>
+        { builder.map((el, i) =>
+            <div className="studentexchange__container" key={i}>
```

Comment:
using index as a key when doing a map is [a react anti-pattern](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9)

Reply:
Updated to element's title instead